.eslintrc.json

```diff
@@ -1,3 +1,24 @@
 {
-  "extends": "amex"
+  "extends": "amex",
+  "overrides": [
+    {
+      "files": ["packages/create-one-app-module/index.js", 
+        "packages/create-one-app-module/create-module.js",
+        "packages/create-one-app-module/**/**"
+      ],
+      "extends": ["amex"],
+      "rules": {
+        "no-console": "off",
+        "unicorn/no-process-exit": "off",
+        "import/no-extraneous-dependencies": "off",
+        "no-empty": "off",
+        "consistent-return": "warn",
+        "no-process-exit": "off"
+      }
+    }, {
+      "files": ["packages/create-one-app-module/__tests__/**"
+      ],
+      "extends": ["amex/test"]
```

Comment:
```suggestion
      "extends": ["amex/prettier/test"]
```

Reply:
There should be a separate effort for this.